### PR TITLE
add local context compose files

### PR DESCRIPTION
**Pull request by sampsapenna**
### Description
Using a local build context for docker-compose makes docker build caching work. This leads to better performance when trying out the complete service stack.

### CODE REVIEW

File: local/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+### Local deployment files for swift-ui stack
+This folder contains the docker-compose files for deploying a complete local
+instance of the swift-ui, with sharing and access request support.
+
+When deployed, the service will be available on port 8080.
+
+The deployment uses the file `envs.txt` as a source for the environment
```
Comment (csc-felipe) [inline local/README.md]: The dockerfiles actually expect a `envs.sh` file.

Comment (sampsapenna) [reply local/README.md]: Fixed

File: local/Dockerfile-postgres-inited [hunk 2]
```diff
@@ -0,0 +1,8 @@
+FROM postgres:9.6-alpine
```
Comment (blankdots) [inline local/Dockerfile-postgres-inited:1]: maybe we should start testing support for postgres 13

Comment (csc-felipe) [reply local/Dockerfile-postgres-inited:1]: Indeed, 9.6 reached end of support last month. https://www.postgresql.org/support/versioning/

**Review by csc-felipe — APPROVED** [review]
This is nice to have, thanks.

File: local/Dockerfile-postgres-inited (hunk 2 above)
Comment (sampsapenna) [reply local/Dockerfile-postgres-inited:1]: I tested with both 13.5 and 14.1 – both work without an issue, but I'll handle that in a separate PR

**Pull request merged**